pagopa/io-app
# feat(IT Wallet): [SIW-1021] Add IT Wallet eID preview screen

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5779, https://github.com/pagopa/io-app/pull/5772

## Short description
This PR adds the eID preview screen with mocked data.

> [!important]
> The screen is missing the component that allows scrolling to the bottom of the page. There is currently an incompatibility with the `IOScrollViewWithLargeHeader` component which will be resolved in a separate PR

## List of changes proposed in this pull request
- Added `ItwIssuanceEidPreviewScreen` component
- Added required locales
- Added mock data for eID, DC, DL and TS in `ItwMockUtils`

## How to test
Navigate to **Profile > IT Wallet > eID > Select an IDP**. You should be able to see the new screen as the below preview.

## Preview

https://github.com/pagopa/io-app/assets/6160324/9d30201f-9c16-4593-8c24-9dab38377389
